### PR TITLE
[To be removed - but first merge other PR originated from dev] updating test output error message in case of failure for no_dash_not_empty tests

### PR DESCRIPTION
**Pull request by pszkamruk-splunk**

### CODE REVIEW

**Comment by harshshah2-crest** [comment]
@pszkamruk-splunk Please update the commit message as per conventional commits (conventionalcommits.org)

**Comment by pszkamruk-splunk** [comment]
I have read the CLA Document and I hereby sign the CLA

File: pytest_splunk_addon/helmut_lib/SearchUtil.py [hunk 1]
```diff
@@ -162,6 +162,13 @@ def checkQueryCountIsZero(self, query, max_time=120):
             self.logger.debug("Count of results is > 0, it is:%d", result_count)
             return False, job.get_results()
 
+    def doSearch(self, query, max_time=20):
```
Comment (harshshah2-crest) [inline pytest_splunk_addon/helmut_lib/SearchUtil.py:165]: - Can we update the function name to get_search_results as function returns the events for the search query.
- Also please add docstring to the function reference: https://github.com/splunk/pytest-splunk-addon/blob/main/pytest_splunk_addon/standard_lib/addon_parser/props_parser.py#L79-L87

Comment (pszkamruk-splunk) [reply pytest_splunk_addon/helmut_lib/SearchUtil.py]: changes done and pushed to the repo
although I am not sure about function name change, all others class methods are following camelCase naming convention

File: pytest_splunk_addon/helmut_lib/SearchUtil.py [hunk 2]
```diff
@@ -162,6 +162,13 @@ def checkQueryCountIsZero(self, query, max_time=120):
...
+        self.logger.debug("query is %s", query)
+
+        job = self.jobs.create(query, auto_finalize_ec=200, max_time=max_time)
+        job.wait(max_time)
+        return job.get_results()
```
Comment (jsowicki-splunk) [inline pytest_splunk_addon/helmut_lib/SearchUtil.py:177]: We should use exception to signal if function output is not complete/malformed.

File: pytest_splunk_addon/standard_lib/fields_tests/test_templates.py [hunk 3]
```diff
@@ -149,9 +150,16 @@ def test_props_fields_no_dash_not_empty(
             record_property("results", results.as_list)
             pp = pprint.PrettyPrinter(indent=4)
             result_str = pp.pformat(results.as_list[:10])
+
+            query_for_unique_events = base_search + " | dedup punct | head 5"
```
Comment (jsowicki-splunk) [inline pytest_splunk_addon/standard_lib/fields_tests/test_templates.py:154]: Use constant naming convention for the query, it's not going to be changed.

Comment (pszkamruk-splunk) [reply pytest_splunk_addon/standard_lib/fields_tests/test_templates.py:154]: code refactored 
added two constants:
TOP_FIVE_STRUCTURALLY_UNIQUE_EVENTS_QUERY_PART = " | dedup punct | head 5"
COUNT_BY_SOURCE_TYPE_SEARCH_QUERY_PART = " | stats count by sourcetype"

File: pytest_splunk_addon/helmut_lib/SearchUtil.py (hunk 2 above)
Comment (pszkamruk-splunk) [reply pytest_splunk_addon/helmut_lib/SearchUtil.py:177]: added try/except to the code which is getting search results

**Comment by bhargav-nariyani-crest** [comment]
@pszkamruk-splunk Can we close this?

**Comment by pszkamruk-splunk** [comment]
@bhargav-nariyani-crest 
Yes, sure. 
Closing PR

**Pull request closed without merging**